packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -188,7 +188,7 @@
 							:disabled="isSaveDisabled"
 							label="Run"
 							icon="pi pi-play"
-							@click="createConfiguration"
+							@click="createConfiguration(false)"
```

Comment:
Why adding the `false` when it is the default value?
```suggestion
							@click="createConfiguration"
```

Reply:
Because linter doesn't like it.